libs/@local/advanced-types/src/omit-value.ts

```diff
@@ -0,0 +1 @@
+export type OmitValue<T, K> = T extends K ? never : T;
```

Comment:
There is an inbuilt TS utility called `Exclude` which does this

Reply:
Ah good point, have removed it in https://github.com/hashintel/hash/pull/4540/commits/53a8c36dce95140a26d168bbbe01a589087bcef9